client/src/components/utils/appHelper.js

```diff
@@ -166,6 +236,13 @@ const toggleInteractionTypeFromMenuClick = (props, interactionTypes) => {
   return props.changeInteractionType(interactionTypes.VIEWING_MAP);
 };
 
+/**
+ * Handle click on Use Current Location card.
+ * @param  {Object}   props            Props passed into the App component
+ * @param  {Object}   interactionTypes Available immutable interaction types
+ * @param  {Function} dispatchFunction Dispatcher that is passed into the action
```

Comment:
Same as above. This is the actual dispatcher used to update the redux store so I think maybe something along the lines of "Redux dispatcher" or "Dispatcher to update redux store"